### DISCUSSION TITLE
点击xmake插件的Build时，弹出的“xmake: config”终端中输出“参数格式不正确 - -Command”，无法启动编译

### DISCUSSION THREAD

**Discussion by ymei-hana** [xmake-io/xmake-vscode#136] [Q&A]
点击xmake插件的Build时，弹出的“xmake: config”终端中出现“参数格式不正确 - -Command”，无法启动编译
另外，点击运行按钮（Run the given target）时也一样出现此输出且无法运行，而点击调试按钮（Debug the given target）时能正常调试

**Comment by waruqi** [comment]
vscode? 截图看看呢

**Reply by ymei-hana** [reply]
> vscode? 截图看看呢

![微信图片_20220902202234](https://user-images.githubusercontent.com/39883695/188141995-f149c8cf-4a43-44f1-8e5d-26b6ae5ae6c0.png)

**Reply by waruqi** [reply]
不清楚，看着不像是 xmake 报错，是插件错误，你可以拉插件源码调下
